### PR TITLE
Drupalcon nola speaking badge

### PR DESCRIPTION
**Pull request by AnneTee**
@chrisarusso Opening a new PR with your first commit to save time.
- Since others might have profile badges (Drupal association?) I added this to the front matter of your team file and set some styling so multiple badges will look nice.
- Updated markdown and tags so Travis will be happy

@kostajh Since chris is out maybe you can glance this over then merge?

### CODE REVIEW

File: _tests/run-tests.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 set -e
 bundle exec jekyll build --config _config.yml,_config.test.yml
-bundle exec htmlproof ./_site --verbose --href-ignore "/^(?:http(?:s)?:\/\/)?(?:[^\.]+\.)?drupal\.org(?:/.*)?$/","/^(?:http(?:s)?:\/\/)?(?:[^\.]+\.)?linkedin\.com(?:/.*)?$/"
+#bundle exec htmlproof ./_site --verbose --href-ignore "/^(?:http(?:s)?:\/\/)?(?:[^\.]+\.)?drupal\.org(?:/.*)?$/","/^(?:http(?:s)?:\/\/)?(?:[^\.]+\.)?linkedin\.com(?:/.*)?$/"
```
Comment (kostajh) [inline _tests/run-tests.sh]: @AnneTee you sure you want to remove it entirely?

Comment (AnneTee) [inline _tests/run-tests.sh]: @kostajh HAH no I just forgot to uncomment, please hold...

**Comment by AnneTee** [comment]
@kostajh Uncommented!

**Comment by kostajh** [comment]
Cool! Nicely done.

**Pull request merged**